include/substrait-mlir/Dialect/Substrait/IR/Substrait.h

```diff
@@ -14,6 +14,7 @@
 #include "mlir/IR/OpImplementation.h"             // IWYU: keep
 #include "mlir/IR/SymbolTable.h"                  // IWYU: keep
 #include "mlir/Interfaces/InferTypeOpInterface.h" // IWYU: keep
+#include "substrait-mlir/Dialect/Substrait/IR/SubstraitEnums.h.inc"
```

Comment:
Sorry, only seeing this now: add a new line before this one, please, and add ` // IWYU: export` at the end. (The lines before are `include`s used by the generated code and the `include`s of the generated files "export" that generated file, so they belong to two different blocks. The `IWYU` pragma is for a tool, `include-what-you-use`, that otherwise wouldn't understand why we include that header since we are not using it. We aren't running that tool currently, so it doesn't matter to much, TBH.)